### ISSUE TITLE
Clearing selected rows

### ISSUE THREAD

**Issue by roytz** [iamisti/mdDataTable#67]
Hi
Is there a way to clear all selected rows (not from ui - but with code)?

**Comment by iamisti** [comment]
No, the concept is not there yet for clearing the selected rows. But good idea, I'll think about it because it's definitely something which would be convenient to use..

**Comment by roytz** [comment]
Thanks.
I saw on the md-data-table.js file that if you pass the value "false" to the "TableDataStorageService.prototype.setAllRowsSelected" function (line 350) it should do it.

**Comment by alan-K-R** [comment]
@iamisti  -- When you use "selectable-rows" with pagination, the header checkbox is set to true on all pages, trying to find a way to clear this on next page..
![image](https://cloud.githubusercontent.com/assets/7842610/21145037/8b53ae66-c12b-11e6-8114-77dc7103f266.png)

 Using $('th md-checkbox').removeClass('md-checked'); on pagination event call but will not work properly:(
